test/e2e/utils/utils.go

```diff
@@ -211,6 +211,23 @@ func TestPodsFileSystemInfo(pods []v1.Pod) error {
 	return nil
 }
 
+func TestInitcontainersRunIdempotent(f *framework.Framework, podList *v1.PodList, pluginName string) error {
```

Comment:
this made me think what we discussed on using the operator in `test/e2e/<device>/*`: when using the operator this whole create / delete / re-create process is much simpler. We can just create and update (the same way how it's done in `test/envtest/*`.

Reply:
Does the 'update' of the operator re run the initcontainer also?

Reply:
`xDevicePlugin`  update triggers an update to the daemonSet. I believe the daemonSet controller then restarts the pods

Reply:
> xDevicePlugin update triggers an update to the daemonSet. I believe the daemonSet controller then restarts the pods

Do I apply this idea to dsa/iaa?
But could you give some info about this?
Idk how to update daemonset in e2e test.

Reply:
Would you mean 'kubectl replace'?
I have no idea how to do the same as envtest..